Script/PRONTO.py

```diff
@@ -1119,6 +1120,24 @@ def update_clinical_tsoppi_file(InPreD_clinical_tsoppi_data_file,sample_id,if_ge
 		fw.close()
 
 
+def pdf_page_image_to_ppt(pdf_file,pages,output_ppt_file):
+	ppt = Presentation(output_ppt_file)
+	slide_width = ppt.slide_width
+	slide_height = ppt.slide_height
+	for page_num in pages:
+		images = convert_from_path(pdf_file, first_page=page_num, last_page=page_num)
+		if not images:
+			print("Warning: The page number " + page_num + " in " + pdf_file + " is not found!")
+			continue
+		slide = ppt.slides.add_slide(ppt.slide_layouts[6])
+		image_path = f"temp_page_{page_num}.jpg"
```

Comment:
just for consistency with other parts of the code, would you mind using this version of format strings:
```suggestion
		image_path = "temp_page_{}.jpg".format(page_num)
```

Reply:
this one is still open